packages/tuono-router/src/components/NotFoundDefaultContent.tsx

```diff
@@ -0,0 +1,12 @@
+import type { JSX } from 'react'
+
+import { Link } from './Link'
+
+export function NotFoundDefaultContent(): JSX.Element {
+  return (
+    <>
+      <h1>Page Not Found</h1>
+      <Link href="/">Return to Homepage</Link>
```

Comment:
This is the only user side relevant change:

| Element | Previous | New |
|--------|--------|--------|
| **Title** | 404 Not found | Page Not Found |
| **Link text** | Return home | Return to Homepage |

> [!NOTE]
> Since is just a wording change I think that it was overkill to create a separate PR, 
> let me know if you prefer I perform this change separately.

Reply:
That is fine to me. Less bureaucracy

Reply:
Looks overkill tho having a separate file for the content. If the reason is only easier testing mocking I would not do it.

Reply:
What do you think about this?

Reply:
I prefer this approach because keeps the default render separated from the pure logic component.